### ISSUE TITLE
TODO: benchmark models

### ISSUE THREAD

**Issue by zdevito** [pytorch/benchmark#5]
- [x] https://github.com/jihunchoi/recurrent-batch-normalization-pytorch/blob/master/bnlstm.py
- [x] LSTMVariants https://github.com/seba-1511/lstms.pth/blob/master/lstms/lstm.py, https://github.com/seba-1511/lstms.pth/blob/master/test/test_speed.py
- [x] (@zdevito) QRNN or SRU https://github.com/taolei87/sru/blob/master/cuda_functional.py https://gist.github.com/jekbradbury/a3a5ae890328db49d8093c1a5bdc8a1e https://openreview.net/pdf?id=H1zJ-v5xl 
- [ ] Yuandong's code (maybe not in the cont. build, but runnable): https://github.com/facebookresearch/ELF/ (compile directory: `./rts/game_MC`, run script: cd ELF
`sh ./train_minirts.sh --gpu 0`)
- [ ] convolution MT (meeting with Michael, David, Sergey to see if it is a good candidate)
- [ ] (@apaszke) tree2tree
- [x] (@ezyang) multiplicative LSTM

Follow up tasks:
- [x] Add batching to  LSTMVariants (some issues in a few layers with mismatched sizes)
- [ ] Annotate all examples with tracer code, and verify correctness.

**Comment by wconstab** [comment]
Cleaning up benchmark repo, closing old issues, please reopen if needed.
